admin-dev/themes/new-theme/js/pages/monitoring/index.js

```diff
@@ -69,12 +67,12 @@ $(() => {
 
     grid.addExtension(new ExportToSqlManagerExtension());
     grid.addExtension(new ReloadListActionExtension());
-    grid.addExtension(new SortingExtension());
```

Comment:
It seems logical to remove them as they are not needed 🤔 did you check @atomiix or do you want us to check that they are not needed for sure ?

Reply:
Yes I checked but I'm not against a double check :)